### PR TITLE
[fix bug 1279557] Fix mapbox maps on profile creation.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: remo/settings/base.py [hunk 1]
```diff
@@ -233,6 +233,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 CSP_SCRIPT_SRC = (
     "'self'",
     "'unsafe-eval'",  # TODO: remove this by pre-compiling handlebars
+    'data:',
```
Comment (johngian) [inline remo/settings/base.py]: Why do you need that?

**Comment by johngian** [comment]
r+wc

File: remo/settings/base.py (hunk 1 above)
Comment (akatsoulas) [inline remo/settings/base.py]: Because mapbox is loading a data URI that is currently rejected by the CSP rules.

**Pull request merged**